app/api/user/[username]/poll/[id]/route.ts

```diff
@@ -0,0 +1,49 @@
+import { findOne, findOneAndUpdate } from "@/utils/user/user";
+import { NextResponse } from "next/server";
+
+export async function GET(
+  request: Request,
+  {
+    params,
+  }: {
+    params: { username: string; id: string };
+  }
+) {
+  try {
+    const username = params.username;
```

Comment:
@Sahil412000 Declare the datatype here and for other variables wherever possible